api/scripts/normalize_all_addresses.py

```diff
@@ -0,0 +1,78 @@
+import asyncio
+import os
+import sys
+
+import asyncclick as click
+from core import logging
+from core.store.database import Database
+from core.util.chain_util import normalize_address
+from core.store.retriever import IntegerFieldFilter
+import pandas as pd
+
+from sqlalchemy.sql.expression import func as sqlalchemyfunc
+from sqlalchemy.sql.expression import or_
+
+sys.path.insert(0, os.path.dirname(os.path.dirname(__file__)))
+from notd.store.schema import TokenTransfersTable
+from notd.store.schema_conversions import token_transfer_from_row
+from notd.store.retriever import Retriever
+from notd.store.saver import Saver
+
+
+@click.command()
+@click.option('-s', '--start-block-number', 'startBlockNumber', required=True, type=int)
+@click.option('-e', '--end-block-number', 'endBlockNumber', required=True, type=int)
+@click.option('-b', '--batch-size', 'batchSize', required=False, type=int, default=1000)
+async def fix_address(startBlockNumber: int, endBlockNumber: int, batchSize: int):
+    databaseConnectionString = Database.create_psql_connection_string(username=os.environ["REMOTE_DB_USERNAME"], password=os.environ["REMOTE_DB_PASSWORD"], host=os.environ["REMOTE_DB_HOST"], port=os.environ["REMOTE_DB_PORT"], name=os.environ["REMOTE_DB_NAME"])
+
+    # databaseConnectionString = Database.create_psql_connection_string(username=os.environ["DB_USERNAME"], password=os.environ["DB_PASSWORD"], host=os.environ["DB_HOST"], port=os.environ["DB_PORT"], name=os.environ["DB_NAME"])
+    database = Database(connectionString=databaseConnectionString)
+    retriever = Retriever(database=database)
+    await database.connect()
+    badOperatorCount = 0
+    badContractCount = 0
```

Comment:
these two need to be moved into the while loop below otherwise you are creating a running total